ocaml/quicktest/quicktest.ml

```diff
@@ -857,7 +859,7 @@ let _ =
     "-iso-sr-path", Arg.String (fun x -> Quicktest_storage.iso_path := x), "Path to ISO SR";
     "-single", Arg.String (fun x -> tests_to_run := [ x ]), Printf.sprintf "Only run one test (possibilities are %s)" (String.concat ", " all_tests) ;
     "-all", Arg.Unit (fun () -> tests_to_run := all_tests), Printf.sprintf "Run all tests (%s)" (String.concat ", " all_tests);
-    "-only-sr", Arg.String (fun x -> Quicktest_storage.only_sr_name := Some x), "Run tests only on SR with specified name";
+    "-default-sr", Arg.Unit (fun () -> use_default_sr := true), "Only run SR tests on the pool's default SR";
```

Comment:
In my opinion it's better if we do it the same way as before, and move the `use_default_sr` ref into `Quicktest_storage`. This way it won't need this parameter, and we won't have to pass it through all the tests that do SR selection.